dev/README.md

```diff
@@ -1,34 +1,54 @@
 
 # Optimus Dev setup
 
-## setup
+## Setup
 + `brew install colima`
 + `make start-colima`
 + check kubernetes context
 + `make apply`
 
+Some optional variable you can set alongside with `make apply`
+```sh
+DAGS_PATH=                # default /tmp/colima/dags
+OPTIMUS_SERVE_PORT=       # default 9100
+OPTIMUS_PLUGIN_ARTIFACTS=
+SETUP_FILE_PATH=          # default ./setup.yaml
+```
+
 ## Components
 + optimus server
 + optimus db (postgres)
 + airflow 
 + airflow db (postgres)
 
 ### Dag file location on your laptop
-+  `/tmp/colima/dags`
-
-### Mounting plugins
-+ upadate `path`'s for `internal-plugins` and `odpf-plugins` volumes in optimus.values.yaml with the actual path to the plugins and upgrade the helm chart for optimus (`make upgrade.optimus`)
-
-```yaml
-volumes:
-  - name: internal-plugins
-    hostPath: 
-      path: /Users/../Documents/proj/optimus-plugins/dist/optimus-plugins_0.6.1_linux_amd64.tar.gz
-      type: File
-  - name: odpf-plugins
-    hostPath: 
-      path: /Users/../Documents/proj/transformers/dist/transformers_0.1.1_linux_arm64.tar.gz
-      type: File
++ `/tmp/colima/dags` or specified by `DAGS_PATH`
+
+### Spinning up the project
++ `mkdir project-a`
++ `cd project-a`
++ `optimus init`
++ `optimus project register`
++ `optimus plugin sync`
++ then load the secret. [ref](#load-secrets)
+
+### Mounting the plugins
++ define plugin artifacts on `setup.yaml` under section `plugins`
++ pass the variable OPTIMUS_PLUGIN_ARTIFACTS when [applying optimus](#setup) (comma separated delimiter). eg.
```

Comment:
why to pass the var when we are listing the plugins in setup.yaml ?

Reply:
updated, committed the new one